### PR TITLE
Workaround for failing circleci tests

### PR DESCRIPTION
**Pull request by KZhu-ME**
Also improved plotting for assembly objects to show amrex grid lines.

### CODE REVIEW

File: mewarpx/mewarpx/emission.py [hunk 1]
```diff
@@ -1397,7 +1397,7 @@ def plot_contours(self):
 
         contours = np.array(skimage.measure.find_contours(inside, 0.17))
 
-        inside = inside.T
+        inside = inside
```
Comment (EtherealMC-Bit) [inline mewarpx/mewarpx/emission.py]: Why do this?

Comment (KZhu-ME) [reply mewarpx/mewarpx/emission.py]: The original plotting function plotted x on the x axis and z on the y axis, but I changed it to plot x on the y axis and z on the x axis to be more consistent with all our other plots.

Comment (peterscherpelz) [reply mewarpx/mewarpx/emission.py]: Pretty sure you can delete the line of code now that you've made the change though, @KZhu-ME :)

Comment (KZhu-ME) [reply mewarpx/mewarpx/emission.py]: Wow, I just realized...

**Review by EtherealMC-Bit — APPROVED** [review]
Looking forward to not getting emails about CircleCI failing (most of the time)

**Review by peterscherpelz — APPROVED** [review]

**Pull request merged**